Realm/RLMFindOptions.mm

```diff
@@ -58,6 +79,18 @@ - (instancetype)initWithProjection:(id<RLMBSON> _Nullable)projection
     return RLMConvertBsonDocumentToRLMBSON(_options.sort_bson);
 }
 
+- (NSArray<RLMSortDescriptor *> *)sortDescriptors {
+    NSMutableArray<RLMSortDescriptor *> *sortDescriptors = [[NSMutableArray alloc] init];
```

Comment:
This conversion logic should be extracted to a function called by each of these types rather than duplicated.